tests/test_datetime_parse.py

```diff
@@ -242,3 +242,8 @@ class Model(BaseModel):
         'type': 'value_error.unicodedecode',
         'msg': "'utf-8' codec can't decode byte 0x81 in position 0: invalid start byte",
     }
+
+
+def test_time_with_timedelta():
```

Comment:
You should put your new test cases [here](https://github.com/samuelcolvin/pydantic/blob/95d6f613b223d532295892090a97952065c1b87a/tests/test_datetime_parse.py#L35)

Reply:
Done. I also added an extra try-except guard when a TZ > 24 hours is stated. 

(I checked parse_datetime which already copes with an error of this nature.)